ios/FluentUI/ActivityIndicator/ActivityIndicatorTokens.swift

```diff
@@ -16,40 +16,21 @@ import UIKit
 }
 
 /// Design token set for the `ActivityIndicator` control.
-public class ActivityIndicatorTokens: ControlTokens {
-    /// Creates an instance of `ActivityIndicatorTokens` with optional token value overrides.
-    /// - Parameters:
-    ///   - size: MSFActivityIndicatorSize enumeration value that will define pre-defined values for side and thickness.
-    ///   - defaultColor: The default color of the Activity Indicator.
-    ///   - side: The value for the side of the square frame of an Activity Indicator.
-    ///   - thickness: The value for the thickness of the ActivityIndicator ring.
-    public init(size: MSFActivityIndicatorSize,
-                defaultColor: DynamicColor? = nil,
-                side: CGFloat? = nil,
-                thickness: CGFloat? = nil) {
-
-        self.size = size
-        super.init()
-
-        // Optional overrides
-        if let defaultColor = defaultColor {
-            self.defaultColor = defaultColor
-        }
-        if let side = side {
-            self.side = side
-        }
-        if let thickness = thickness {
-            self.thickness = thickness
-        }
+open class ActivityIndicatorTokens: ControlTokens {
+    /// MSFActivityIndicatorSize enumeration value that will define pre-defined values for side and thickness.
+    public var size: MSFActivityIndicatorSize {
+        return state.size
     }
 
-    let size: MSFActivityIndicatorSize
+    unowned var state: MSFActivityIndicatorState!
```

Comment:
Using and unowned var to point back to the state seems dangerous as it's tightly coupled to the lifetime of the state object.

Why can't we just store the style/size properties directly in the tokens object and query or set them through the state object?

Reply:
Done!